sktime/forecasting/ar.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+
+__author__ = ["Markus Löning"]
+__all__ = ["AutoAR"]
+
+from statsmodels.tsa.ar_model import ar_select_order
+
+from sktime.forecasting.base._statsmodels import _StatsModelsAdapter
```

Comment:
Throwing this error while running: No module named 'sktime.forecasting.base._statsmodels'

Reply:
The `_StatsModelsAdapter` class still exists, it moved into a different location. If you search for it, you'll find it! `sktime/forecasting/base/_adapters.py` perhaps?